code/datums/mutations/speech.dm

```diff
@@ -120,27 +120,49 @@
 	var/message = speech_args[SPEECH_MESSAGE]
 	if(message)
 		message = " [message] "
+		message = replacetext(message," security ",pick(" coppers "," bobbies "))
+		message = replacetext(message," friend ",pick(" mate "," bruv "," lad "))
+		message = replacetext(message," great ",pick(" bangin' "," sound "))
+		message = replacetext(message," man ",pick(" mate "," mans "))
+		message = replacetext(message," isnt ",pick(" innit "," aint it "))
+		message = replacetext(message," dumb ",pick(" daft "," stupid "))
+		message = replacetext(message," idiot ",pick(" wanker "," tosser "," mong "," prick "))
+		message = replacetext(message," little ",pick(" tad "," wee "))
+		message = replacetext(message," credits ",pick(" pounds "," quid "))
+		message = replacetext(message," drunk ",pick(" shitfaced "," pissed "))
+		message = replacetext(message," fucking "," bloody ")
+		message = replacetext(message," dick "," knob ")
+		message = replacetext(message," dickhead "," bellend ")
+		message = replacetext(message," pussy "," fanny ")
+		message = replacetext(message," stupid "," daft ")
 		message = replacetext(message," looking at  ","  gawpin' at ")
-		message = replacetext(message," great "," bangin' ")
-		message = replacetext(message," man "," mate ")
-		message = replacetext(message," friend ",pick(" mate "," bruv "," bledrin "))
+		message = replacetext(message," head "," dome ")
+		message = replacetext(message," bar "," pub ")
+		message = replacetext(message," water "," wa'er ")
+		message = replacetext(message," music "," tunes ")
+		message = replacetext(message," dissapointed "," gutted ")
+		message = replacetext(message," vacuum "," hoover ")
+		message = replacetext(message," sus "," dodgy ")
+		message = replacetext(message," sleep "," kip ")
+		message = replacetext(message," tired "," knackered ")
+		message = replacetext(message," fries "," chips ")
+		message = replacetext(message," chips "," crisps ")
 		message = replacetext(message," what "," wot ")
-		message = replacetext(message," drink "," wet ")
-		message = replacetext(message," get "," giz ")
-		message = replacetext(message," what "," wot ")
-		message = replacetext(message," no thanks "," wuddent fukken do one ")
-		message = replacetext(message," i don't know "," wot mate ")
-		message = replacetext(message," no "," naw ")
 		message = replacetext(message," robust "," chin ")
-		message = replacetext(message,"  hi  "," how what how ")
-		message = replacetext(message," hello "," sup bruv ")
-		message = replacetext(message," kill "," bang ")
-		message = replacetext(message," murder "," bang ")
+		message = replacetext(message," what "," wot ")
```

Comment:
Uh, isn't this done 2 lines above already?

Reply:
Yep, my bad. Fixed now